src/Checkbox.js

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'react-emotion';
+import { rem } from 'polished';
+
+import Box from './Box';
+import Flex from './Flex';
+import Text from './Text';
+import Icon from './Icon';
+
+const CheckIcon = ({ fill = 'currentColor', ...props }) => (
```

Comment:
Should these icon's be living in fort awesome?

Reply:
These should be extracted, but it was feeling like a blocker to getting this out at the time. Still not completely clear on the advantages of icon fonts over svgs, but I know they exist...

Reply:
The biggest reason is that inline SVG's cannot be cached and rendering them requires a new DOM tree for each instance. If you're only showing it once, an external SVG is probably the better call. For something reused a customized font-kit is the best option.